src/cryptonote_config.h

```diff
@@ -157,6 +157,7 @@ static_assert(STAKING_PORTIONS % 12 == 0, "Use a multiple of twelve, so that it
 #define HF_VERSION_ED25519_KEY                  cryptonote::network_version_13_enforce_checkpoints
 #define HF_VERSION_FEE_BURNING                  cryptonote::network_version_14_blink
 #define HF_VERSION_BLINK                        cryptonote::network_version_14_blink
+#define HF_VERSION_MIN_2_OUTPUTS                cryptonote::network_version_16
```

Comment:
I thought this was already enforced.  Or is it just done by the wallet but not enforced?

Reply:
Just wallet, it would fail on construction, but if you tweak it like we do with LNS transactions then it goes through fine. (1 change output + tx extra burn fee).